README.md

```diff
@@ -6,17 +6,12 @@ for working with buckets, tables and workspaces are covered.
 
 ## Install
 
-`$ pip3 install git+https://github.com/keboola/sapi-python-client.git`
-
-or 
-
 ```bash
-$ git clone https://github.com/keboola/sapi-python-client.git && cd sapi-python-client
-$ python setup.py install
+pip install git+https://github.com/keboola/sapi-python-client.git
 ```
 
 ## Client Class Usage
-```
+```python
```

Comment:
just curious, what does this do?

Reply:
uses the python highligher, for simple code it does not matter, as the generic one does well, but sometimes it does some weirdness